dmd-config/dmd.conf

```diff
@@ -1,5 +1,5 @@
 [Environment32]
-DFLAGS=-I%@P%/../import/druntime -I%@P%/../import/phobos -L-L%@P%/../lib32 -L--export-dynamic
+DFLAGS=-I%@P%/../import/druntime -I%@P%/../import/phobos -L-L%@P%/../lib32 -L--export-dynamic -L-rpath=/snap/dmd/current/lib32 -L--enable-new-dtags
```

Comment:
Can you explain what the `--enable-new-dtags` flag does and why it needs to be added here?  (Not just for me, but in the commit message as well.)